templates/index.tpl

```diff
@@ -25,8 +25,8 @@
                         {t}Audio only (MP3){/t}
                     </label>
                     <div class="seekOptions">
-                        {t}From{/t} <input type="text" placeholder="00:00:00" value="" name="from" />
-                        {t}to{/t} <input type="text" placeholder="00:22:30" value="" name="to" />
```

Comment:
I'm starting to think this placeholder is a bit confusing, it does not tell users if they need to use minutes or seconds.
Maybe with could use something like `placeholder="MM:SS"`.